src/lexer.coffee

```diff
@@ -116,10 +117,18 @@ exports.Lexer = class Lexer
     if id is 'from' and @tag() is 'YIELD'
       @token 'FROM', id
       return id.length
-    if id is 'as' and @seenImport and (@tag() is 'IDENTIFIER' or @value() is '*')
-      @tokens[@tokens.length - 1][0] = 'IMPORT_ALL' if @value() is '*'
-      @token 'AS', id
-      return id.length
+    if id is 'as' and @seenImport
+      # loop hack
```

Comment:
Those tests pass in 1.11.1. They should still pass after your PR. I didn’t test the code I put in the comment, it was just meant to pose a question about the `loop`. I don’t like the idea of `loop`, which becomes `while (true)`. I feel like whatever you’re trying to do here, you should be able to do with a simple `if`/`else` tree. Is there a reason we need a loop?